mobile/lib/features/student_management/presentation/pages/create_student_page.dart

```diff
@@ -0,0 +1,115 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../../domain/entities/student.dart';
+import '../bloc/student_bloc.dart';
+
+class AddClass extends StatefulWidget {
+  @override
+  _AddClassState createState() => _AddClassState();
+}
+
+class _AddClassState extends State<AddClass> {
+  TextEditingController _firstnameController = TextEditingController();
+  TextEditingController _lastnameController = TextEditingController();
+  TextEditingController _idController = TextEditingController();
+  TextEditingController _classroomIdController = TextEditingController();
+  StudentBloc _studentBloc;
+  int _levelSelected;
+  final List<int> levels = [
+    1,
+    2,
+    2,
+    4,
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    //WELP ME PLIS
+    _studentBloc = BlocProvider.of<StudentBloc>(context);
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('New student'),
+      ),
+      body: Padding(
```

Comment:
Please, try to refactor and extract the widgets from this `ListView`.

Also, I can see you only use Containers to add padding. I think a better solution is to apply padding to the `ListView` (16.0 for example) and only apply padding to the elements if necessary.